orbs/phraseapp/.phraseapp-sample.yml

```
@@ -0,0 +1,12 @@
+# Example of configuration file
+
+phraseapp:
+  available_locales:
+    en
+    fr
+    de
+  pull:
+    targets:
+      - file: ./config/locales/
+        params:
+          file_format: yml
```

```suggestion
          # recommended: use yml for Ruby project, nested_json for JS, go_i18n for Go
          file_format: yml
```

Based on what we use in jobteaser, ui-jobteaser and services